src/index.cpp

```diff
@@ -292,7 +292,6 @@ namespace diskann {
         if (_pts_to_labels.size() > 0) {
           std::ofstream label_writer(std::string(filename) + "_labels.txt");
           for (_u32 i = 0; i < _pts_to_labels.size(); i++) {
```

Comment:
So what is the new file format for filters? Could you please document this as comments somewhere in the code?

Optionally (not required for closing this PR) Consider writing a worksflow for building, computing GT, and searching filtered index in the ROOT/workflows/ directory. You could use the workflow guides there as examples.

Reply:
the label file format I have mentioned in the comments. 
Workflows - will be including in a separate PR